src/components/new-simulation/simulation-results.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import { useCommonState } from "../../hooks/use-common-state";
+import { useAppContext } from "../../hooks/use-app-context";
+import { SimulationGraphsCheckboxes } from "./simulation-graphs-checkboxes";
+import { SimulationGraph } from "./simulation-graph";
+
+import "./simulation-results.scss";
+
+interface ISimulationResultsProps {
+  control1: boolean;
+  control2: boolean;
+  simulationTime: number;
+  experimentsRun: boolean[][];
+}
+
+export function SimulationResults({ control1, control2, simulationTime, experimentsRun }: ISimulationResultsProps) {
+  const ac = useAppContext();
+  const { getAllExperiments } = useCommonState(ac);
+
+  // Which checkboxes are checked
+  const [graphsShowing, setGraphsShowing] = useState([[false, false], [false, false]]);
+  graphsShowing[+control1][+control2] = true;
```

Comment:
I'm not sure if this matters, but it feels weird to me to mutate state right after declaring it. I would probably do something like this:
```suggestion
  const gs = [[false, false], [false, false]];
  gs[+control1][+control2] = true;
  const [graphsShowing, setGraphsShowing] = useState(gs);
```

Reply:
You are right it is a bit weird. 
Changing it to a one-time initialization failed to automatically select the new graph when the user switches controls.  I ended up with a `useEffect` which makes things nicely explicit.